src/QirRuntime/test/Standalone/InputTypes/generate.py

```diff
@@ -0,0 +1,45 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Is this script used anywhere in CI builds? If so, I'd suggest migrating it to PowerShell; we've consolidated the rest of our build scripts to use PowerShell rather than having a mix of bash, cmd, and PowerShell.

Reply:
This file is no longer needed since Irina recently added support for [generating .ll files from a Q# project](https://github.com/microsoft/qsharp-runtime).

I'll remove this file.

Reply:
Removed.